thanos/values.yaml

```diff
@@ -143,6 +144,57 @@ query:
     minAvailable: 1
     maxUnavailable: ""
 
+# EXPERIMENTAL COMPONENT
+# Thanos receive configuration
+receive:
+  enabled: true
```

Comment:
I think this should be disabled by default...

Reply:
Yep, it was forgotten on my part

Reply:
fixed